Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -61,7 +61,7 @@
       },
       {
         "Name": "OwnershipBasedAuthorization",
-        "IsEnabled": false
+        "IsEnabled": true
```

Comment:
@semalaiappan I spoke with @vimayya and she suggested leaving this feature off in the base _appsettings.json_ file, but turning it on in _appsettings.development.json_ instead.

Reply:
I updated it

Reply:
@vimayya We apparently didn't have an _appsettings.development.json_ file until now. You okay with us adding one now?

Reply:
appsettings.development.json doesn't exist in the repo but is created by the initdev powershell in Invoke-NewDevelopmentAppSettings

Reply:
added in  initdev powershell in Invoke-NewDevelopmentAppSettings process